juspay/hyperswitch
# fix(connector): [Stripe] capture error message and error code for failed payment, capture, void and refunds

## Type of Change



## Description

Resolves [#3737](https://github.com/juspay/hyperswitch-cloud/issues/3737)




## How did you test it?

This following change effects error response handling of auth, refunds, void, and capture.  However for certain calls and payment method. We can't produce an error scenario. Hence, to ensure the changes doesn't break any flow, do the following testing.
1. Create a Bank Transfer payment with stripe. If possible try to fail it. 
2. Create a capture payment and capture it.
3. Create a capture payment and void it.
4. Create and confirm a payment and refund it.
5. Verify all payment methods with capture


## Checklist
